src/Platforms/AbstractPlatform.php

```diff
@@ -393,6 +396,7 @@ abstract public function getName();
      *
      * @param string $dbType
      * @param string $doctrineType
+     * @psalm-param non-empty-string $doctrineType
```

Comment:
I'm not sure if we need this here. A type assertion on the `Types\Type::hasType()` should do the job. If that method doesn't throw, we know we have a valid Doctrine type (which by definition is non-empty).

Reply:
`Types\Type::hasType()` doesn't throw on a type named `''`.